[BUG] Building fails with BUILD_TESTING=1

**Describe the bug**
When enabling the building of tests, the build fails due to missing header files

**Steps to reproduce**
1. Clone this repository
2. Configure with tests enabled:

        mkdir build && cmake -DBUILD_TESTING=1 ..

4. Build:

        make

**Expected behavior**
Build finishes successfully

**Actual behavior**
Build fails with:

```
[ 50%] Building CXX object test/CMakeFiles/socket_test.dir/socket_test.cpp.o
/home/sander/aslanx/src/sd_sockets/test/socket_test.cpp:18:10: fatal error: sd_sockets/sd_sockets.hpp: No such file or directory
   18 | #include "sd_sockets/sd_sockets.hpp"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

This is due to include directories not set on the test target.

--- a/test/CMakeLists.txt
+++ b/test/CMakeLists.txt
@@ -1,5 +1,9 @@
 add_executable(socket_test socket_test.cpp)
 target_link_libraries(socket_test GTest::gtest GTest::gtest_main)
+target_include_directories(
+  socket_test
+  PRIVATE ${CMAKE_SOURCE_DIR}/include
+)
 
 include(GoogleTest)
 gtest_discover_tests(socket_test)

